packages/ui-plugin-sheets/src/controllers/shortcuts/utils.ts

```diff
@@ -15,3 +21,11 @@ export function whenEditorActivatedIsVisible(contextService: IContextService) {
         contextService.getContextValue(FOCUSING_EDITOR) && contextService.getContextValue(FOCUSING_EDITOR_BUT_HIDDEN)
     );
 }
+
+export function whenEditorFormulaActivated(contextService: IContextService) {
```

Comment:
Renaming these functions and context keys to clarify them.

Reply:
> Renaming these functions and context keys to clarify them.

use `whenEditorInputFormulaActivated`, `whenEditorDidNotInputFormulaActivated`, and context key `FOCUSING_EDITOR_INPUT_FORMULA`